dualscreeninfo/package.json

```diff
@@ -36,20 +36,19 @@
     "@babel/runtime": "*",
     "react": "*",
     "react-native": "*",
-    "react-native-windows":  "*"
+    "react-native-windows": "*"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "@react-native-community/bob": "^0.9.7",
     "@types/jest": "^25.1.1",
-    "@types/react-native": "^0.61.10",
+    "@types/react": "^16.8.6",
```

Comment:
>"@types/react": "^16.8.6", [](start = 4, length = 26)

I think this should be updated to 16.9.0

Reply:
fixed